### DISCUSSION TITLE
[Feature] Facial recognition merge same name/different face detection

### DISCUSSION THREAD

**Discussion by Nonobis** [immich-app/immich#2494] [Feature Request]
### The feature

Hello,

First, great work on facial recognition ! It's working really great, i love this :)

I have found a little bug i think, i have faces from same person a different age (baby, child, adult) ... facial recognition detect a different person for different age.
 
Could you associate the image in same facial recognition when we tag face with a name ? Same name => merging display/recognition ?

Thank again for the great work !

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by ShinasShaji** [comment]
According to a collaborator, this is planned, but not implemented yet. See [discussion](https://github.com/immich-app/immich/discussions/2468#discussioncomment-5943230).

**Comment by jrasm91** [comment]
Closing in favor of #2472,  where we can include this in the list of potential enhancements.
